modules/k6/k6.go

```diff
@@ -17,17 +21,78 @@ type K6Container struct {
 	testcontainers.Container
 }
 
+type RemoteTestFileDescription struct{
+
+	Uri string
```

Comment:
I'd choose a typed URL here, so the user does not commit any mistake building it.